libavcodec/qsv_internal.h

```diff
@@ -90,6 +90,7 @@ typedef struct QSVSession {
 #ifdef AVCODEC_QSV_LINUX_SESSION_HANDLE
     AVBufferRef *va_device_ref;
     AVHWDeviceContext *va_device_ctx;
+    void *loader;
```

Comment:
typedef struct QSVSession {
    mfxSession session;
#ifdef AVCODEC_QSV_LINUX_SESSION_HANDLE
    AVBufferRef *va_device_ref;
    AVHWDeviceContext *va_device_ctx;
    void *loader;
#endif
} QSVSession;

Should be:

typedef struct QSVSession {
    mfxSession session;
    void *loader;
#ifdef AVCODEC_QSV_LINUX_SESSION_HANDLE
    AVBufferRef *va_device_ref;
    AVHWDeviceContext *va_device_ctx;
#endif
} QSVSession;

Otherwise on Windows:

CC      libavcodec/qsv.o
C:/msys64/home/Artem/FFMPEG-new-session/libavcodec/qsv.c: In function 'ff_qsv_init_internal_session':
C:/msys64/home/Artem/FFMPEG-new-session/libavcodec/qsv.c:566:41: error: 'QSVSession' has no member named 'loader'
  566 |                                      &qs->loader);
      |                                         ^~
C:/msys64/home/Artem/FFMPEG-new-session/libavcodec/qsv.c: In function 'ff_qsv_close_internal_session':
C:/msys64/home/Artem/FFMPEG-new-session/libavcodec/qsv.c:992:21: error: 'QSVSession' has no member named 'loader'
  992 |         MFXUnload(qs->loader);
      |                     ^~
C:/msys64/home/Artem/FFMPEG-new-session/libavcodec/qsv.c:994:11: error: 'QSVSession' has no member named 'loader'
  994 |         qs->loader = NULL;
      |           ^~
make: *** [/home/Artem/F